api/all/src/main/java/io/opentelemetry/api/trace/ArrayBasedTraceState.java

```diff
@@ -21,6 +21,9 @@
   @Override
   @Nullable
   public String get(String key) {
+    if (key == null) {
```

Comment:
Is this one really necessary? Looks like it'll just fall through and return null as currently implemented, no?

Reply:
Not really necessary, nah, but it makes it more obvious that we gracefully handle null instead of having to verify the implementation